compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -670,10 +679,11 @@ static void PrintService(const ServiceDescriptor* service,
   (*vars)["service_name"] = service->name();
   (*vars)["file_name"] = service->file()->name();
   (*vars)["service_class_name"] = ServiceClassName(service);
+  (*vars)["grpc_version"] = XSTR(GRPC_VERSION);
```

Comment:
Having  "version: unknown".  That looks like a bug.  If the version isn't defined, it shouldn't even be in the message, as in none of " (version $grpc_version$)"